global.json

```diff
@@ -0,0 +1,5 @@
+{
+    "sdk": {
+      "version": "5.0.x"
```

Comment:
I'm personally more a fan of pinning an exact SDK version as I originally had [here](https://github.com/G-Research/consuldotnet/blob/434e400a9dda7d8a367a0996e53d303db672235f/global.json). Azure agents update on their own timeline with no warning, and it's also clearer for contributors. That's a moving minor release range though, so acceptable.

Reply:
I think it is reasonable trade-off. Pinning the exact version can be annoying for individual developers as it requires to install that exact version os the SDK even if they already have newer version installed.

Reply:
As you prefer :-)